packages/financial-templates-lib/src/logger/Transports.ts

```diff
@@ -80,14 +83,17 @@ export function createTransports(transportsConfig: TransportsConfig = {}): Trans
         )
       );
     }
+
+    // to disable pdv2, pass in a "disabled=true" in configs or env. Configs get merged rather than replaced as was the
+    // convention in previous transports.
+    const { disabled: pagerDutyV2Disabled = false, ...pagerDutyV2Config } = {
+      ...JSON.parse(process.env.PAGER_DUTY_V2_CONFIG || "null"),
+      ...transportsConfig.pagerDutyV2Config,
```

Comment:
Maybe I'm misunderstanding, but from what I see, it doesn't look like the configs get merged in other configs, it looks like we try to find it in the transportsConfig first and then we try to parse it from the env. I think this behavior is reasonable as long as there's a good reason for us to prefer it and for this config to act differently from the others.

1. Where does the transportsConfig come from under typical circumstances?
2. Is this merging process needed?

Reply:
1. I think under typical circumstances it will come from a config file
2. The configs do not get merged in the previous pattern, you suggested that this does not need to maintain bw compatibility with previous method so i was proposing a different way.  Let me see what i can do about keeping it in line with the previous convention, but the additional `disabled` key makes the typing tricky.